src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -85,6 +139,43 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             auto scheme = *it;
             ColorSchemeComboBox().SelectedItem(scheme);
         }
+
+        // populate color table grid
+        const auto colorLabelStyle{ Resources().Lookup(winrt::box_value(L"ColorLabelStyle")).as<Windows::UI::Xaml::Style>() };
+        const auto colorControlStyle{ Resources().Lookup(winrt::box_value(L"ColorControlStyle")).as<Windows::UI::Xaml::Style>() };
+        const auto colorTableEntryTemplate{ Resources().Lookup(winrt::box_value(L"ColorTableEntryTemplate")).as<DataTemplate>() };
+        auto setupColorControl = [colorTableEntryTemplate, colorControlStyle, colorTableGrid{ ColorTableGrid() }](hstring colorName, auto&& colorRef, uint32_t row, uint32_t col) {
+            ContentControl colorControl{};
+            colorControl.ContentTemplate(colorTableEntryTemplate);
+            colorControl.Style(colorControlStyle);
+            ToolTipService::SetToolTip(colorControl, box_value(colorName));
+            Automation::AutomationProperties::SetName(colorControl, colorName);
+
+            Data::Binding binding{};
+            binding.Source(colorRef);
+            binding.Mode(Data::BindingMode::TwoWay);
+            colorControl.SetBinding(ContentControl::ContentProperty(), binding);
```

Comment:
These are all things I don't think we had examples of doing in code before. This is good to have as a reference, thanks ☺️